features/support/feature_tests/action/paypal.rb

```diff
@@ -0,0 +1,112 @@
+# coding: utf-8
+module FeatureTests
+  module Action
+    module Paypal
+      extend Capybara::DSL
+      extend RSpec::Matchers
+
+      module_function
+
+      def connect_marketplace_paypal
+        topbar = FeatureTests::Section::Topbar
+        paypal_preferences = FeatureTests::Section::MarketplacePaypalPreferences
+        admin_sidebar = FeatureTests::Section::AdminSidebar
+
+        # Connect Paypal for admin
+        topbar.navigate_to_admin
+        admin_sidebar.click_payments_link
+        paypal_preferences.connect_paypal_account
+
+        expect(page).to have_content("PayPal account connected")
+
+        # Save payment preferences
+        paypal_preferences.set_payment_preferences("2.0", "5", "1.0")
+        paypal_preferences.save_settings
+        expect(page).to have_content("Payment preferences updated")
+      end
+
+      def connect_seller_paypal
+        topbar = FeatureTests::Section::Topbar
+        settings_sidebar = FeatureTests::Section::UserSettingsSidebar
+        paypal_preferences = FeatureTests::Section::UserPaypalPreferences
+
+        # Connect Paypal for seller
+        topbar.open_user_menu
+        topbar.click_settings
+        settings_sidebar.click_payments_link
+        paypal_preferences.connect_paypal_account
+
+        expect(page).to have_content("PayPal account connected")
+
+        # Grant commission fee
+        paypal_preferences.grant_permission
+
+        expect(page).to have_content("Hooray, everything is set up!")
+      end
+
+      def request_listing(listing_title)
+        home = FeatureTests::Page::Home
+        listing = FeatureTests::Page::Listing
+        listing_book = FeatureTests::Page::ListingBook
+
+        home.click_listing(listing_title)
```

Comment:
This was the only actions that didn't start with the topbar that is visible in all pages, so I just added the logo click as you suggested.

That adds an extra request that wouldn't be needed for this test case, but maybe we can take that hit to avoid weird problems in the future?